style.css

```diff
@@ -121,13 +120,16 @@ section {
   justify-content: space-between;
   padding: 1rem;
 }
+
 .left{
   align-items: start;
 }
+
 .right{
   width: 100;
   align-items: end;
 }
+
 .create{
   color: #be3144;
   margin: 1rem;
```

Comment:
- [ ] Please Kindly leave one space always before opening a curly braces of a block statement. This will ensure that the rule which enforces consistent spacing inside braces of object literals etc is met.